packages/components/src/LkFieldTree/stories/FieldPicker.story.tsx

```diff
@@ -54,17 +48,13 @@ const FieldGroupHeading = (props: ParagraphProps) => (
 const fields = (
   <>
     <TreeBranch>
-      <FieldGroupHeading>DIMENSIONS</FieldGroupHeading>
+      <FieldGroupHeading pt="0">DIMENSIONS</FieldGroupHeading>
```

Comment:
```suggestion
      <FieldGroupHeading pt="none">DIMENSIONS</FieldGroupHeading>
```

Should this just be baked in to `FieldGroupHeading`?

Reply:
Explore actually wants the first heading to NOT have any `pt` (probably since the actual accordion content container has that `0.5rem` of vertical padding in their View-level Trees). Note that the "Filter-Only Fields" heading appears to have `pt` but really it's just the `pt` from the Tree style override.
![Screen Shot 2021-08-23 at 2 30 22 PM](https://user-images.githubusercontent.com/16812885/130522080-7d619060-20f8-474c-8378-ff112e7b6d83.png)

Wanted to note that our `FieldGroupHeading` actually isn't exported as component (it's just a styled `<p>` used in the story). That being said, I _could_ create / export a `LkFieldHeading` if we wanted to do something like that.

Reply:
Ah - that makes sense, I don't see much sense in export `FieldGroupHeading` :)